features/glance/src/main/java/com/escodro/glance/presentation/TaskListGlanceWidget.kt

```diff
@@ -27,114 +28,149 @@ import androidx.glance.appwidget.provideContent
+         * background color and corner radius.
+         */
+        val background = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            GlanceModifier.cornerRadius(10.dp).background(GlanceTheme.colors.background)
+        } else {
+            GlanceModifier.background(ImageProvider(R.drawable.rounded_corners))
+        }
+
+        AlkaaGlanceTheme {
+            Column(
+                modifier = GlanceModifier.appWidgetBackground().fillMaxSize().then(background)
+                    .padding(8.dp),
             ) {
-                Image(
-                    provider = ImageProvider(R.drawable.ic_alkaa_icon),
-                    contentDescription = "",
-                    modifier = GlanceModifier.size(32.dp)
-                        .clickable(actionStartActivity(getHomeIntent())),
-                )
-            }
-            if (taskList.isEmpty()) {
-                EmptyListContent()
-            } else {
-                TaskListContent(taskList = taskList)
+                Row(
+                    modifier = GlanceModifier.fillMaxWidth().height(32.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    Image(
+                        provider = ImageProvider(R.drawable.ic_alkaa_icon),
+                        contentDescription = "",
+                        modifier = GlanceModifier.size(32.dp)
+                            .clickable(actionStartActivity(getHomeIntent())),
+                    )
+                    Spacer(modifier = GlanceModifier.width(12.dp))
+                    Text(
+                        text = context.getString(R.string.glance_heading),
+                        style = TextStyle(
+                            color = GlanceTheme.colors.onSurface,
+                            fontSize = 16.sp,
+                            fontWeight = FontWeight.Medium,
+                        ),
+                        modifier = GlanceModifier.defaultWeight(),
+                    )
+                }
+                if (taskList.isEmpty()) {
+                    EmptyListContent()
+                } else {
+                    TaskListContent(taskList = taskList)
+                }
             }
         }
     }
 
     @Composable
-    private fun EmptyListContent() {
-        Box(modifier = GlanceModifier.fillMaxSize().padding(top = 16.dp)) {
-            Text(text = "No tasks")
+    private fun EmptyListContent(
+        modifier: GlanceModifier = GlanceModifier,
+        context: Context = LocalContext.current,
+    ) {
+        Column(
+            modifier = modifier.fillMaxSize(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Image(
+                provider = ImageProvider(R.drawable.ic_task_list),
+                contentDescription = "Blank Task List",
```

Comment:
Please, add this string to the `strings.xml`. The Portuguese translation is "Lista de Tarefas Vazia".